src/librustdoc/html/render/mod.rs

```diff
@@ -901,8 +903,13 @@ fn render_assoc_item(
             }
         };
         let vis = meth.visibility.print_with_space(meth.def_id, cx).to_string();
-        let constness =
-            print_constness_with_space(&header.constness, meth.const_stability(cx.tcx()));
+        // FIXME: Once https://github.com/rust-lang/rust/issues/67792 is merged, we can remove
+        // this condition.
+        let constness = if render_mode.is_normal() {
+            print_constness_with_space(&header.constness, meth.const_stability(cx.tcx()))
+        } else {
+            ""
+        };
```

Comment:
My whole point was that there should be an exhaustive pattern match *here*; it's still unclear what the `else` branch is referring to. Can you get rid of `is_normal` and inline the match please?

Reply:
In the two locations?

Reply:
Yes.